src/Components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import '../CSS/Header.css'
+
+function Header() {
+  return (
+    <header>
+      <img src='./assets/tomato.jpg' alt='tomato' />
```

Comment:
We'll have to look into a possible file path issue with this image, since it's currently not displaying.